### PR TITLE
adding Versions section to readme

### PR DESCRIPTION
**Pull request by SomkaPe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Windows Azure Active Directory Authentication Library (ADAL) for Node.js
 The ADAL for node.js library makes it easy for node.js applications to authenticate to AAD in order to access AAD protected web resources.  It supports 3 authentication modes shown in the quickstart code below.
 
+## Versions
+Current version - 0.1.22  
+Minimum recommended version - 0.1.22
```
Comment (polita) [inline README.md]: @weijjia Is there really only one recommended/supported version?

Comment (weijjia) [inline README.md]: adal node is still under preview version, we should always recommend the devloper use the latest version. @addev-ashish correct me if i'm wrong.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Windows Azure Active Directory Authentication Library (ADAL) for Node.js
 The ADAL for node.js library makes it easy for node.js applications to authenticate to AAD in order to access AAD protected web resources.  It supports 3 authentication modes shown in the quickstart code below.
```
Comment (jeffwilcox) [inline README.md]: Is "Windows Azure" even a brand name to be used vs. "Azure Active Directory"?

Comment (polita) [inline README.md]: @jeffwilcox Good point. Can I ask you to open a separate issue for that?

Comment (jeffwilcox) [inline README.md]: Issue/PR here: https://github.com/AzureAD/azure-activedirectory-library-for-nodejs/pull/136

**Comment by addev-ashish** [comment]
:shipit:

**Comment by polita** [comment]
:shipit:

**Comment by jeffwilcox** [comment]
I think you guys have the merge PR button, I don't ;-)

**Pull request merged**